### PR TITLE
Add writeup on Ember Engines

### PR DESCRIPTION
**Pull request by amyrlam**
@villander Let me know what you think, anything to add or edit?

### CODE REVIEW

File: source/2020-05-22-the-ember-times-issue-149.md [hunk 1]
```diff
@@ -25,14 +26,12 @@ READMORE
...
-## [Section title in sentence case 🐹](section-url)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [Ember Engines acceptance testing guides 📝](https://ember-engines.com/docs/testing-acceptance)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[Michael Villander (@villander)](https://github.com/villander) and team have fleshed out docs about the **acceptance test story** in the Ember Engines ecosystem, touching on some bleeding-edge cases. Check out the new acceptance testing guides [here](https://ember-engines.com/docs/testing-acceptance). Special thanks to [Dan Gebhardt (@dgeb)](https://github.com/dgeb), [Gabriel Csapo (@gabrielcsapo)](https://github.com/gabrielcsapo) and [Thomas Gossmann (@gossi)](https://github.com/gossi) for their reviews and tips!
```
Comment (villander) [inline source/2020-05-22-the-ember-times-issue-149.md]:
```suggestion
[Michael Villander (@villander)](https://github.com/villander) and team have fleshed out docs about the **acceptance test story** in the Ember Engines ecosystem, touching on some bleeding-edge cases. Also, the entire doc examples were migrated to Ember Octane. Check out the new acceptance testing guides [here](https://ember-engines.com/docs/testing-acceptance). Special thanks to [Dan Gebhardt (@dgeb)](https://github.com/dgeb), [Gabriel Csapo (@gabrielcsapo)](https://github.com/gabrielcsapo) and [Thomas Gossmann (@gossi)](https://github.com/gossi) for their reviews and tips!
```

File: source/2020-05-22-the-ember-times-issue-149.md [hunk 2]
```diff
@@ -25,14 +26,12 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [Ember Engines acceptance testing guides 📝](https://ember-engines.com/docs/testing-acceptance)
```
Comment (villander) [inline source/2020-05-22-the-ember-times-issue-149.md]:
```suggestion
## [Ember Engines acceptance testing guides & Octane 📝](https://ember-engines.com/docs/testing-acceptance)
```

**Comment by villander** [comment]
I left some suggestions ; )

**Comment by amyrlam** [comment]
@villander thank you for your help! 🙇‍♀️

File: source/2020-05-22-the-ember-times-issue-149.md [hunk 3]
```diff
@@ -25,14 +26,12 @@ READMORE
...
-## [Section title in sentence case 🐹](section-url)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [Ember Engines acceptance testing guides & Octane 📝](https://ember-engines.com/docs/testing-acceptance)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[Michael Villander (@villander)](https://github.com/villander) and team have fleshed out docs about the **acceptance test story** in the Ember Engines ecosystem, touching on some bleeding-edge cases. Also, the entire doc examples were migrated to Ember Octane! Check out the new acceptance testing guides [here](https://ember-engines.com/docs/testing-acceptance). Many thanks to [Dan Gebhardt (@dgeb)](https://github.com/dgeb), [Gabriel Csapo (@gabrielcsapo)](https://github.com/gabrielcsapo) and [Thomas Gossmann (@gossi)](https://github.com/gossi) for their reviews and tips!
```
Comment (ijlee2) [inline source/2020-05-22-the-ember-times-issue-149.md]: To my understanding, linking the word "here" isn't a good accessibility practice. Can we change the sentence to something like:

```suggestion
[Michael Villander (@villander)](https://github.com/villander) and team have fleshed out docs about the **acceptance test story** in the Ember Engines ecosystem, touching on some bleeding-edge cases. Also, the entire doc examples were migrated to Ember Octane! Visit the official site to see [Ember Engines' new acceptance testing guides](https://ember-engines.com/docs/testing-acceptance). Many thanks to [Dan Gebhardt (@dgeb)](https://github.com/dgeb), [Gabriel Csapo (@gabrielcsapo)](https://github.com/gabrielcsapo) and [Thomas Gossmann (@gossi)](https://github.com/gossi) for their reviews and tips!
```

**Review by ijlee2 — APPROVED** [review]
Looks good. Thanks!

File: source/2020-05-22-the-ember-times-issue-149.md (hunk 3 above)
Comment (amyrlam) [reply source/2020-05-22-the-ember-times-issue-149.md]: @ijlee2 TIL, thank you!
